rb/poll.py

```diff
@@ -157,6 +157,15 @@ def poll(self, timeout=None):
         return rv
 
 
+def is_closed(f):
+    poller = poll()
```

Comment:
Yeah, it's kinda shitty, but I don't feel like duplicating all the logic in poll to optimize this. The pollers are not that expensive anyways in the grand scheme of things.